packages/sdk/src/trackers/visitorId.ts

```diff
@@ -1,15 +1,28 @@
 import { Trackable } from '../Trackable';
-import { uuid, Cookie } from '../utils';
+import { Cookie, uuid } from '../utils';
 
 const COOKIE_NAME = 'cnfdVisitorId';
 
-export const visitorIdentity = (): Trackable.Manager => controller => {
-  if (typeof document === 'undefined') return;
-  let value = Cookie.get(COOKIE_NAME);
-  if (!value) {
-    value = uuid();
-    // TODO check correct cookie options
-    Cookie.set(COOKIE_NAME, value, { maxAge: 60 * 60 * 24 * 365 * 5 });
+export const visitorIdentity =
+  (clientSecret: string): Trackable.Manager =>
+  controller => {
+    if (typeof document === 'undefined') return;
+    let value = Cookie.get(COOKIE_NAME);
+    if (!value) {
+      value = uuid();
+      // TODO check correct cookie options
+      Cookie.set(COOKIE_NAME, value, { maxAge: 60 * 60 * 24 * 365 * 5 });
+    }
+
+    const hashedValue = generateUniqueId(value + clientSecret).toString();
+    controller.setContext({ visitor_id: hashedValue });
```

Comment:
Let's put the hashed value as the cookie value so we have the same visitor id stored in the cookie and in the context.

This will enable us to guide the customer to access the Cookie value from the requests and thus make it easier to use the same visitor ID both backend and frontend resolutions.